README.md

```diff
@@ -0,0 +1,29 @@
+<p align="center">
+<img src="src/frontend/static/icons/Hipster_HeroLogoCyan.svg" width="300"/>
+</p>
+Demo: Deploy application micorservices on Kubernetes in Azure AKS with Istio
```

Comment:
You can say instead: Build, deploy and run a microservices-based online boutique application using Pulumi, Helm, and Istio.
We will add the other technos later (ArgoCD, Keptn, Prometheus,...)